docs/administering/ssl.md

```diff
@@ -46,7 +46,8 @@ To share port 443 with other services on the same machine:
   web server on port 443** with Sandstorm.
   
 - You [can follow this guide](https://web.archive.org/web/20190922195059/https://juanjoalvarez.net/es/detail/2017/jan/12/how-set-sandstorm-behind-reverse-proxy-keeping-you/)
-  that explains how to use a [cron script](https://github.com/juanjux/sandstorm-sandcats-cert-installer) 
-  to extract the certificates from your (sandcats.io enabled) Sandstorm installation to a location and 
-  format where your reverse proxy can use them so it can serve Sandstorm by HTTPS, keeping your 
-  sandcats.io domain and free auto-renewable certificates, along with any other services on your server.
+  that explains how to use a [cron script](https://github.com/Michael-S/sandstorm_certs_extract_cron)
+  to extract the certificates from your Sandstorm-managed TLS Sandstorm installation. Please note that the
```

Comment:
Possibly pedantic, but "installation with Sandstorm-managed TLS" would read better and be less repetitive.